.travis.yml

```diff
@@ -2,6 +2,11 @@ arch: amd64
 os: linux
 dist: focal
 
+branches:
+  only:
+    - master
+    - release/*
```

Comment:
I'm just not sure if globbing is the accepted nomenclature here.  I saw some indications of regex.  Couldn't find anything definitive though.

Reply:
yeah, not sure either.